manim/mobject/geometry.py

```diff
@@ -402,9 +387,25 @@ def __init__(self, point=ORIGIN, **kwargs):
 
 
 class SmallDot(Dot):
-    CONFIG = {
-        "radius": DEFAULT_SMALL_DOT_RADIUS,
-    }
+    CONFIG = {"radius": DEFAULT_SMALL_DOT_RADIUS}
+
+
+class AnnotationDot(VMobject):
+    def __init__(self, color=BLUE):
+        VMobject.__init__(self)
```

Comment:
Question: Shouldn't `**kwargs` come in both the `__init__`s? I may have missed some recent updates about CONFIG. Did something change which doesn't require `**kwargs`?

Reply:
I am not sure, maybe some else can give insides, @leotrs ?

Reply:
Yes, otherwise you can't use stuff like `color=RED` when instantiating a new `AnnotationDot`.